packages/csv-parser-price/src/main.js

```diff
@@ -148,13 +155,19 @@ export default class CsvParserPrice {
   // eslint-disable-next-line class-methods-use-this
   mergeBySku(data, currentPrice) {
     const previousPrice = data.prices[data.prices.length - 1]
-    const sku = CONSTANTS.header.sku
-    if (previousPrice && previousPrice.sku === currentPrice[sku])
-      previousPrice.prices.push(currentPrice)
+
+    // remove variant-sku property from price object
+    const {
+      [CONSTANTS.header.sku]: currentSku,
+      ..._currentPrice
```

Comment:
This naming confused me a little. Maybe `restOfCurrentPrice`?

Just something that would make it easier to understand difference between `currentPrice` and `_currentPrice`.